db/migrate/20150206201847_add_site_id_to_storytime_post.rb

```diff
@@ -0,0 +1,11 @@
+class AddSiteIdToStorytimePost < ActiveRecord::Migration
+  def change
+    add_column :storytime_posts, :site_id, :integer
+
+    site = Storytime::Site.first
```

Comment:
This (and the other ones adding site_id) are going to break when someone sets up a brand new site and calls rake db:migrate for the first time, since they will have no site set up yet. 

For all of these, we should wrap the assignment of the site id in a check to verify a site already exists.